lib/trento/application/projectors/sap_system_projector.ex

```diff
@@ -149,6 +152,21 @@ defmodule Trento.SapSystemProjector do
     end
   )
 
+  project(
+    %SapSystemUpdated{
```

Comment:
Does this need an after update broadcast?

Reply:
Yes, coming in a new PR with frontend work